modules/javafx.graphics/src/main/native-glass/mac/GlassMenu.m

```diff
@@ -262,6 +262,22 @@ - (void)_setPixels:(jobject)pixels
         (*env)->CallVoidMethod(env, pixels, jPixelsAttachData, ptr_to_jlong(&image));
         if (image != NULL)
         {
+            jclass pixelsClass = (*env)->FindClass(env, "com/sun/glass/ui/mac/MacPixels");
+
+            jfieldID jPixelsWidthField = (*env)->GetFieldID(env, pixelsClass, "width", "I");
+            jfieldID jPixelsHeightField = (*env)->GetFieldID(env, pixelsClass, "height", "I");
+            jfieldID jPixelsScaleXField = (*env)->GetFieldID(env, pixelsClass, "scalex", "F");
+            jfieldID jPixelsScaleYField = (*env)->GetFieldID(env, pixelsClass, "scaley", "F");
```

Comment:
As Kevin already commented, these four fields can be cached and set in `initIDs`.